sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -1728,6 +1728,7 @@ def test_queue_message_http_proxy_setting(self, uamqp_transport):
         assert receiver._config.http_proxy == http_proxy
         assert receiver._config.transport_type == TransportType.AmqpOverWebsocket
 
+    @pytest.mark.skip("Disable Until Accurate Link Detach Handling Implemented")
```

Comment:
Detach Error causes a Infinite Loop on Message Settlement because we currently do not handle the case of DETACH_SENT to the service (addressing this in a separate PR to ensure the impact of change is not widespread)

Reply:
That seems like a release blocking problem - anyway we can make it known that main is in an unshippable state until that follow-up PR arrives? I think we have an issue tag for that?